front/pages/api/w/[wId]/labs/transcripts/default.ts

```diff
@@ -1,13 +1,20 @@
 import type { WithAPIErrorReponse } from "@dust-tt/types";
+import { isLeft } from "fp-ts/lib/Either";
+import * as t from "io-ts";
 import type { NextApiRequest, NextApiResponse } from "next";
 
 import { Authenticator, getSession } from "@app/lib/auth";
 import { LabsTranscriptsConfigurationResource } from "@app/lib/resources/labs_transcripts_resource";
 import { apiError, withLogging } from "@app/logger/withlogging";
 import type { GetLabsTranscriptsConfigurationResponseBody } from "@app/pages/api/w/[wId]/labs/transcripts";
+import { acceptableTranscriptProvidersCodec } from "@app/pages/api/w/[wId]/labs/transcripts";
 
 const PROVIDERS_WITH_WORKSPACE_CONFIGURATIONS = ["gong"];
 
+export const GetDefaultTranscriptsConfigurationBodySchema = t.type({
+  provider: acceptableTranscriptProvidersCodec,
```

Comment:
Weird that io-ts does not you to use `t.string("gong")`. Are you sure this works?

Reply:
IIRC io-ts does not support array out of the box, you need to pass through a union type of string literals.

Reply:
Yeah that's what I'm doing :) 

```
export const acceptableTranscriptProvidersCodec = t.union([
  t.literal("google_drive"),
  t.literal("gong"),
]);
```

Reply:
Seems like the `PROVIDERS_WITH_WORKSPACE_CONFIGURATIONS` const on the line above confused you, it's not the one used in this line